.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
+docstring-convention = numpy
+import_order_style = smarkets
+max-line-length = 88
+extend-ignore = E203,I202,I100
```

Comment:
Just curious, what is I100? Do you know why I didn't need these exclude in reduceSPI?

Reply:
It's very similar as I202, with the spacing between the imports. Flake does not know that ioSPI is local, and wants to put the imports together, while isort knows and splits them up. See https://github.com/compSPI/ioSPI/pull/19/commits/2d7e03c2d619bcc9caabc177e0907bc2940eeb40

In your tests you do https://github.com/compSPI/reduceSPI/blob/master/tests/test_offline.py#L3, but there is no other import like `import numpy` or something nearby. My guess is that you would have the same issue.